### PR TITLE
Add language dropdown for verifying webhook signature

### PR DESCRIPTION
**Pull request by LearningNerd**
Adds a language dropdown with code snippet for Node as well as Python to the webhooks page:
https://docs.sentry.io/workflow/integrations/integration-platform/webhooks/#verifying-the-signature

Do we want to add other language examples too? :)

### CODE REVIEW

File: src/collections/_documentation/workflow/integrations/integration-platform/webhooks-verifying-signature/node.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+```javascript
+const crypto = require('crypto');
+
+function verifySignature(request, secret='') {
+  const hmac = crypto.createHmac('sha256', secret);
+  hmac.update(JSON.stringify(request.body), 'utf8');
+  const digest = hmac.digest('hex');
+  return digest === request.headers['sentry-hook-signature'];
```
Comment (MeredithAnya) [inline src/collections/_documentation/workflow/integrations/integration-platform/webhooks-verifying-signature/node.md]: I think we need this to be capitalized to `'Sentry-Hook-Signature'`

Comment (LearningNerd) [reply src/collections/_documentation/workflow/integrations/integration-platform/webhooks-verifying-signature/node.md]: Hm in Express all the headers were coming back lower-case but yeah I'll change it :)

Comment (LearningNerd) [reply src/collections/_documentation/workflow/integrations/integration-platform/webhooks-verifying-signature/node.md]: @MeredithAnya whoops I lost track of this, could you check this latest update and approve / merge? :) thanks!

**Comment by LearningNerd** [comment]
Ping @MeredithAnya I forgot about this one -- it still says merging is blocked and lists you as the pending reviewer / changes requested, and I had made the changes you suggested. :) If you can accept/finish the review that'd be awesome. Thanks!

**Pull request merged**